frontEnd/src/pages/Home.tsx

```diff
@@ -1,3 +1,59 @@
+import { v4 as uuidv4 } from 'uuid';
+import { useNavigate } from 'react-router-dom';
+import Button from '@/components/Button';
+import useInput from '@/hooks/useInput';
+
 export default function Home() {
-  return <div>Home</div>;
+  const navigate = useNavigate();
+  const { inputValue, onChange } = useInput('');
+
+  const handleMakeRoomClick = () => {
+    const roomId = uuidv4();
+    navigate(`/${roomId}`);
+  };
+
+  const handleJoinRoomClick = () => {
+    if (inputValue) navigate(`/${inputValue}`);
+  };
+
+  const handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === 'Enter') handleJoinRoomClick();
+  };
+
+  return (
+    <div className="h-[100vh] w-[100vw] bg-backgroundColor flex justify-center items-center py-[10vw] ">
```

Comment:
`[100vh]` 대신 `h-screen`으로 사용할 수 있을 것 같아요!!
```suggestion
    <div className="h-screen w-screen bg-backgroundColor flex justify-center items-center py-[10vw] ">
```